HackYourFuture-CPH/Guide-IT
# migrations for questions table

# Description

Created Migrations for Questions Table #28
Fixes # (issue)
#28 
# How to test?

Please run npm run db:setup then you can verify the questions table in the database.

# Checklist

- [x ] I have performed a self-review of my own code
- [x ] This PR is ready to be merged and not breaking any other functionality
